lib/commands/doctor/checks/install.js

```diff
@@ -37,16 +37,6 @@ const tasks = {
         });
     },
     nodeVersion: function nodeVersion(ctx) {
-        const globalBin = execa.shellSync('npm bin -g', {preferLocal: false}).stdout;
```

Comment:
Can you explain why this removal is part of this commit? 🤔

Reply:
The reason why we needed that check in the first place was because sqlite is node version specific.

Because we're using the knex-migrator installed with Ghost itself (and hence the sqlite version installed there), this check isn't necessary anymore.

Reply:
Oh i see, thanks 👍